hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -32,7 +32,7 @@
 
 	public void run(HookContext hookContext) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)